modules/upgrade/pages/client-x.adoc

```diff
@@ -115,43 +100,6 @@ PXE booting is not used during this process.
 Approximately three minutes after the job was fetched, the client will go down for reboot.
 
 
-////
```

Comment:
I think we want to enable this now because we now must cover the SLE 15 to SLE 15 SP1 upgrade etc. Maybe, to be moved in a client-y file.

Reply:
Sure